src/IssueViz.Security/HotspotsList/TableDataSource/HotspotTableEntry.cs

```diff
@@ -53,7 +54,7 @@ public bool TryGetValue(string keyName, out object content)
                     content = hotspot.RuleKey;
                     break;
 
-                case StandardTableColumnDefinitions.Priority:
+                case PriorityTableColumnDefinition.ColumnName:
                     content = hotspot.Priority.ToString();
```

Comment:
I don't know how I missed this, `StandardTableColumnDefinitions.Priority` only accepts values of [VSTASKPRIORITY](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.interop.vstaskpriority?view=visualstudiosdk-2019) enum, and it's an icon! I don't know how it worked for me, I think I used a different column to test this, then changed to priority and didn't re-test 😞